src/test/java/org/apache/sysds/test/functions/builtin/BuiltinBayesianOptimisationTest.java

```diff
@@ -1,3 +1,22 @@
+/*
```

Comment:
Please reformat the test file. Add some test cases for different parameters of the optimizer such as number of iterations, minimize, and ExecType (CP or SPARK).